__tests__/app.test.js

```diff
@@ -99,3 +99,15 @@ describe("GET comments", ()=>{
     })
 });
 
+describe("POST comments", ()=>{
+    test("Returns status code of 201 - created. Responds with the posted commment.",()=>{
+        return request(app)
+        .post("/api/articles/1/comments")
+        .send({username : "lurker", body : "unsubing rn"})
+        .expect(201)
+        .then((responce) => {
+            expect(responce.body).toHaveProperty("username")
+            expect(responce.body).toHaveProperty("body")
+        })
+    })
+});
```

Comment:
We need to handle the edge cases too. 
- what if the body is missing
- if the username doesn't exist
- if the article doesn't exist
- if the article_id is invalid